data/systemd/snapd.core-fixup.sh

```diff
@@ -11,6 +11,11 @@ if ! grep -q "ID=ubuntu-core" /etc/os-release; then
     exit 0
 fi
 
+# No fixe-ups yet on UC20
+if [ -e /var/lib/snapd/modeenv ]; then
```

Comment:
BTW. is this the only way to distinguish uc20 system?

Reply:
We could also check "grep -q snapd_recovery_mode=r /proc/cmdline" for example.

Reply:
I think the cmdline check is a bit better but probably okay for a followup

Reply:
+1 on the cmdline check, or a check against /writable itself

Reply:
I will do that too in a followup, the reason is that I tested this approach and because testing is manual it's "expensive" to change it. But I hope I can change this soon(ish).